src/providers/emoji/Emojis.cpp

```diff
@@ -219,6 +221,10 @@ void Emojis::loadEmojis()
                 parseEmoji(variationEmojiData, variation,
                            emojiData->shortCodes[0] + "_" + toneName);
 
+                // NOTE: After parsing an emoji variation, it's category gets set to ""
+                // We have to manually inherit the category from the original emojiData.
```

Comment:
Makes it sound like we're taking/moving the category.

```suggestion
                // NOTE: Emoji variations don't have a "catgory".
                // We have to manually inherit the category from the original emojiData.
```